app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
```

Comment:
`.split()` method works the same by default. If u are not passing anything iside - it will split by space

```suggestion
    command = command.split()
```

Reply:
that comment left